### PR TITLE
Updated contact details for collaboration

### PR DESCRIPTION
**Pull request by jonk-ebi**
A minor change to the content to correct a missing update for  [ENSWEB-6783 Paul Flicek leaving](https://www.ebi.ac.uk/panda/jira/browse/ENSWEB-6783)

### CODE REVIEW

**Review by jyothishnt — APPROVED** [review]

File: htdocs/info/about/collaborations/index.html [hunk 1]
```diff
@@ -13,20 +13,15 @@ <h1>Collaborations</h1>
 short intensive collaborative project. 
 <a href="/info/about/outreach/geek4aweek.html">More information</a>...</p>
 
-<p>
-Formal requests for, say, large scale collaborations, legal issues, etc., are best sent to 
-<a href="http://www.ebi.ac.uk/about/people/paul-flicek" rel="external">Paul Flicek</a>
-at the EMBL-EBI.
-</p>
+<p>Formal requests such as large scale collaborations, legal issues, etc., are best sent to Rob Finn and/or Andy Yates at EMBL-EBI.</p>
```
Comment (imransl) [inline htdocs/info/about/collaborations/index.html:16]: Links to the people/person pages in EMBL-EBI for Rob Finn and Andy Yates are missing here

Comment (jonk-ebi) [reply htdocs/info/about/collaborations/index.html]: Good catch, just added them

**Review by imransl — APPROVED** [review]

File: htdocs/info/about/collaborations/index.html (hunk 1 above)
Comment (jyothishnt) [reply htdocs/info/about/collaborations/index.html]: Should these be email addresses (mailTo)?

Comment (jonk-ebi) [reply htdocs/info/about/collaborations/index.html]: The linked pages contains their e-mail. If we link to that page we won't have to update this again should their e-mail addresses change. They also give some context to who they are which might be useful to whomever wishes to contact them

Comment (jyothishnt) [reply htdocs/info/about/collaborations/index.html]: OK Cool

**Pull request merged**